src/components/goalDetail/GoalBasket.tsx

```
@@ -7,11 +7,11 @@ import {
 import DatePicker from 'react-datepicker';
 import 'react-datepicker/dist/react-datepicker.css';
 
-type Props = {
+interface Props {
   basketItems: { id: number; task: string }[];
   onPickDate: (index: number, date: Date | null) => void;
```

ESLint

[error] 'index' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)